drivers/include/net/netdev2.h

```diff
@@ -20,13 +20,19 @@
  *
  * Example call flow:
  *
- * 1. packet arrives -> driver calls netdev2->event_callback with
- *    event==NETDEV_EVENT_ISR (from interrupt routine)
- * 2. event_callback wakes up upper layer thread
- * 3. upper layer calls netdev2->driver.isr()
- * 4. netdev2->driver.isr() calls netdev2->event_callback() with
- *    event==NETDEV_EVENT_RX_COMPLETE
- * 5. netdev2->event_callback() uses netdev2->driver.recv() to fetch packet
+ * 1. packet arrives for device
+ * 2. driver calls @ref netdev2_t::event_callback "netdev->event_callback()"
+ *    with `event` := @ref NETDEV2_EVENT_ISR (from Interrupt Service Routine)
+ *    and wakes up event handler
+ * 3. event handler calls @ref netdev2_driver_t::isr "netdev2->driver->isr()"
```

Comment:
(The `@ref struct::member "link-text"` must be used, otherwise doxygen won't display the links correctly)